### ISSUE TITLE
Pressure issue w/ the tablet monitor

### ISSUE THREAD

**Issue by Nayro** [drawpile/Drawpile#395]
Hey guys. Hopefully this is the right place to post this.

I recently got a Yiynova MVP22U (a Cintiq alternative) to replace my Wacom Intuos4. I had no issues with pressure in Drawpile prior, but now whenever I draw, no matter how fast or slow, I get this:
![documents](https://cloud.githubusercontent.com/assets/25441458/22440682/43fafd16-e73d-11e6-9bca-22c8bb11c5af.jpg)

Things I tried: updating Drawpile, reinstalling Drawpile, turning off Digital Ink in the tablet settings, fiddling around with Input settings... I have experienced no issues whatsoever in Photoshop or SAI and my drivers are all up to date, which leads me to believe that this is a software bug.

I found a similar issue here, but it looks a bit different from mine. I didn't find any solution. https://github.com/callaa/Drawpile/issues/296

Any help would be appreciated, I was looking forward to using Drawpile again D:

**Comment by callaa** [comment]
Most likely another driver related incompatibility. After many updates, Qt's built-in tablet support is still very broken.

Krita has worked around the problem by writing their own tablet library. I've tried copying it once before and managed to get it almost working. The sense that the program compiled correctly, but the new code didn't do anything. What I was missing then was that it requires a patched version of Qt to work.

I'm now investigating a new build system that would make it possible/easier to build the patched version of Qt for Windows. If it works, I may be able to integrate Krita's tablet code which will hopefully fix these glitches.

**Comment by Nayro** [comment]
Both of my friends who I use Drawpile with use Cintiq's, so I was hoping I wouldn't run into any problems either. 

I guess what you said also explains why I can't get my tablet to work in FireAlpaca or the older versions of OpenCanvas. I also tried Krita out of curiosity -  it's EXTREMELY buggy. I can barely get past the resolution screen (some driver bug message that duplicates itself like a hundred times), drawing is near impossible. Welp. Guess they really don't like Yiynova.

I'm not holding my breath on it being fixed soon, but who knows. Thank you for your response, @callaa !

**Comment by callaa** [comment]
If Krita doesn't support it, then sadly their tablet library will not help in this case. (Although I believe it will improve support for some other tablets.) After skimming the tablet handling code, I got the impression that most of the problems are caused by terrible tablet drivers and the workarounds developers have to put in to support their quirks. Wacom stuff usually works best.

**Comment by Nayro** [comment]
The absolute worst part is that Drawpile actually works near flawlessly... if it wasn't for those infuriatingly random dots. But anyway, I understand now. I'll try pinging Yiynova support about this, they're usually pretty snappy with their fixes and driver updates... Usually being the key word.

Thank you again! I guess you might as well close this topic now.

**Comment by Nayro** [comment]
Hey @callaa !

I wanted to add that, as I recently found out, the issue I was experiencing with the newest version of Krita actually did not exist prior to 3.0. The issue is apparently universal for all non-Wacom tablet monitors, since both Ugee and Yiynova users are reporting it.

Here's the bug thread describing the issue: https://bugs.kde.org/show_bug.cgi?id=363284
(the resolution screen bug I mentioned was caused by extended desktop, so ignore that one)

I've no idea if it helps at all, but I thought I'd mention it.
